packages/rocketchat-livechat/.app/client/views/register.js

```diff
@@ -114,4 +120,15 @@ Template.register.onCreated(function() {
 		$('.error').removeClass('show');
 		this.error.set();
 	};
+
+	this.guestName = new ReactiveVar(null);
+	this.autorun(() => {
+		this.guestName.set(Livechat.guestName);
```

Comment:
any reason to not use `Livechat.guestName` directly on helper instead of creating this reactive var?